xchem_db/xchem_db/settings.py

```diff
@@ -1,50 +1,46 @@
 """
-Django settings for test_xchem project.
+Django settings for xchem_db project.
 
-Generated by 'django-admin startproject' using Django 2.0.4.
+Generated by 'django-admin startproject' using Django 3.1.5.
 
 For more information on this file, see
-https://docs.djangoproject.com/en/2.0/topics/settings/
+https://docs.djangoproject.com/en/3.1/topics/settings/
 
 For the full list of settings and their values, see
-https://docs.djangoproject.com/en/2.0/ref/settings/
+https://docs.djangoproject.com/en/3.1/ref/settings/
 """
 
+from pathlib import Path
 import os
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
 
 
 # Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'u4mgin$vk3g!q%lq4vqp(u3g9=pyy5x95cosp)+%rfh%$p#z%y'
+SECRET_KEY = '7!%=0w@rmjou1bo63u81$57u8^x%snfz&*11@l&3+&&824cm5e'
```

Comment:
Should we hide this secret key from being published here on github?

Maybe the whole settings file should be tracked elsewhere than with the source?